typescript/infra/src/config/chain.ts

```diff
@@ -20,6 +21,32 @@ import { getSecretRpcEndpoints } from '../agents/index.js';
 
 import { DeployEnvironment } from './environment.js';
 
+// Separate list of chains that we want to keep updated operationally but
```

Comment:
might be worth mentioning why we need legacy ones, iirc it was block gas limits for the new ICA router?